.devcontainer/devcontainer.json

```diff
@@ -11,7 +11,8 @@
     "ms-python.vscode-pylance",
     "visualstudioexptteam.vscodeintellicode",
     "redhat.vscode-yaml",
-    "esbenp.prettier-vscode"
+    "esbenp.prettier-vscode",
+    "kdcro101.favorites"
```

Comment:
I don't want this extension personally...

IMHO we need to keep this to a functional level, not taste.

Reply:
I agree with @frenck

Reply:
ok, these were very fast and concrete responses 🤔